ThumbnailItem/ThumbnailItem.js

```diff
@@ -40,25 +99,46 @@ const ThumbnailItemBase = kind({
 		publicClassNames: true
 	},
 
-	render: ({children, css, src, ...rest}) => {
+	computed: {
+		className: ({roundThumbnail, styler}) => styler.append({roundThumbnail}),
+
+		subComponents: ({content, subContent, css}) => {
```

Comment:
> In FORTECH-37 for agate/ImageItem, the children prop is used for the content. I think that it would be better to match APIs among components and among theme libraries. In another theme library, we uses the children prop for the Content and label for the Sub Content. So my suggestion is...Let's rename content and subContent as children and label.

According to my comment above, we can rename `subComponents` as `children`. Then the `children` will contain the `children` for content and the `label` for subContent.